evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -150,6 +154,59 @@ pub(crate) fn set_registers_and_run<F: RichField>(
     interpreter.run()
 }
 
+/// Computes the JUMPDEST proofs for each context.
+///
+/// # Arguments
+///
+/// - `jumpdest_table_rpc`:  The raw table received from RPC.
+/// - `code_db`: The corresponding database of contract code used in the trace.
+pub(crate) fn set_jumpdest_analysis_inputs_rpc(
+    jumpdest_table_rpc: &JumpDestTableWitness,
+    code_map: &HashMap<H256, Vec<u8>>,
+) -> JumpDestTableProcessed {
+    let ctx_proofs = (*jumpdest_table_rpc)
+        .iter()
+        .flat_map(|(code_addr, ctx_jumpdests)| {
+            let code = if code_map.contains_key(code_addr) {
+                &code_map[code_addr]
+            } else {
+                &vec![]
+            };
+            trace!(
+                "code: {:?}, code_addr: {:?} <============",
+                &code,
+                &code_addr
+            );
+            trace!("code_map: {:?}", &code_map);
```

Comment:
Do we want to keep this?

Reply:
I want to keep all tracing until we are happy with the computed jumpdest tables.  It doesn't takes longer to re-add it than to remove it.